.github/workflows/set-backlog-fields.yml

```
@@ -0,0 +1,103 @@
+name: Add Issue to Project
+
+on:
+  issues:
+    types: [ opened ]
+
+jobs:
+  steve:
```

```suggestion
  add-issue-to-board:
```
"steve" doesn't sound like the right job name to me :)

---

I dunno, I think that job name would apple, I mean apply, to a lot of situations.